Soccar [Game-Server]/src/nl/soccar/gameserver/model/command/CommandListener.java

```diff
@@ -23,7 +24,7 @@ public CommandListener(Privilege minimumPrivilege, boolean hostOnly) {
         this.hostOnly = hostOnly;
     }
 
-    public abstract void execute(PlayerWrapper player, RoomWrapper room, Command command);
+    public abstract void execute(PlayerWrapper player, Chat chat, Command command);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)